modules/MinimumSpanningTree/cover.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Ok. That works. If it's easier for you, I have no problems with **(A):** `git rm`-ing the file on my end, rr, if you prefer, I can **(B):** altogether eliminate the file from the history of this branch and force push to update the pull request. Of course, I don't want to rewrite history if you or another Gephi team member have already pulled this branch down somewhere and started tinkering with it :) Please let me know if I should do either **(A)** or **(B)** otherwise I will assume you will exclude it from the merge.